.github/workflows/test-build.yaml

```diff
@@ -0,0 +1,23 @@
+name: Test build WASM version
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [wasm]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the Repository
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+      - uses: actions/setup-python@v3
+        with:
+          python-version: "3.x"
+
+      - name: Build the WASM version
+        run: |
+          pip install --upgrade pip
+          make deploy
```

Comment:
Should we use `make -B deploy` here as well, as in the build CI?

Reply:
I'll add it

Reply:
Would it be okay if I add a `<h1>&chi;iplot</h1>` to `index.html` (above "Initialising the WebDash environment")?

Reply:
Yes, that’s absolutely ok